src/sentry/workflow_engine/endpoints/utils/test_fire_action.py

```diff
@@ -35,3 +44,43 @@ def test_fire_action(
             action_exceptions.append(f"An unexpected error occurred. Error ID: '{error_id}'")
 
     return action_exceptions
+
+
+def get_test_notification_event_data(project) -> GroupEvent | None:
```

Comment:
This might be a nice to have method in the `grouptype.py` file -- so it's easy to import the issue type / the way to generate the fake event. (incase another endpoint or feature needs it in the future).